### PR TITLE
I18n 310

### PR DESCRIPTION
**Pull request by josuepr2**
Added new functions on  backends/yaml/yaml.go:

1. **NewWithPacker** function that read yml configuiration file for i18n give a folder path

2. **NewWithPackerAndWalk** function that read yml configuiration file for i18n give a folder path. That function as a plus iterate all the subfolders in the given path.

### CODE REVIEW

File: backends/yaml/yaml.go [hunk 1]
```diff
@@ -66,6 +93,30 @@ func NewWithWalk(paths ...string) i18n.Backend {
...
+// New new YAML backend for I18n that implements backer to package all the files in the directory
+func NewWithPackerAndWalk(paths ...string) (*Backend, error){
+
+	backend := &Backend{}
+
+	for _, path := range paths {
+		box := packr.New("i18nConfig", path)
+		filesInBox := box.List()
+
+		log.Println(filesInBox)
```
Comment (CarlosPadilla) [inline backends/yaml/yaml.go]: Remove logs

File: backends/yaml/yaml.go [hunk 2]
```diff
@@ -44,6 +46,31 @@ func New(paths ...string) *Backend {
...
+	backend := &Backend{}
+
+	for _, path := range paths {
+		box := packr.New("i18nConfig", path)
+		filesInBox := box.List()
+
+		for _, file := range filesInBox {
+			if !strings.Contains(file, "/") && strings.Contains(file, ".yml") {
+
+				log.Println("********* ", file)
```
Comment (CarlosPadilla) [inline backends/yaml/yaml.go]: Remove logs

**Review by CarlosPadilla — CHANGES_REQUESTED** [review]

File: backends/yaml/yaml.go (hunk 1 above)
Comment (josuepr2) [reply backends/yaml/yaml.go]: done

File: backends/yaml/yaml.go (hunk 2 above)
Comment (josuepr2) [reply backends/yaml/yaml.go]: done

**Review by CarlosPadilla — APPROVED** [review]

**Pull request merged**